httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/MultipartEntityBuilder.java

```diff
@@ -245,20 +286,20 @@ MultipartFormEntity buildEntity() {
         final AbstractMultipartFormat form;
         switch (modeCopy) {
             case LEGACY:
-                form = new LegacyMultipart(charsetCopy, boundaryCopy, multipartPartsCopy);
+                form = new LegacyMultipart(charsetCopy, boundaryCopy, multipartPartsCopy, preamble, epilogue);
```

Comment:
@arturobernalg Looks good to me. I would not add preamble and epilogue in the LEGACY mode, though.

Reply:
Changed.